docs/concepts/pdp/configuration.mdx

```diff
@@ -264,6 +264,31 @@ Default: `<green>{time}</green> | {process} | <blue>{name: <40}</blue>|<level>{l
 
 Change this parameter to change the log format (e.g. remove colors).
 
+#### PDP_DATA_UPDATE_CALLBACKS
+
+Default: `[]`
+
+List of data update callbacks to be called when data is updated. For example:
+```json
+[
+  {
+    "url": "http://example.com/callback",
+    "key": "example",  // optional
+    "config": {  // optional
```

Comment:
this isn't a valid json

Reply:
Right, it's docs it does not have to be valid 🙂